examples/wolf_sheep/wolf_sheep/agents.py

```diff
@@ -1,6 +1,5 @@
 import random
-
-from mesa import Agent
+from mesa.EvoAgent import *
```

Comment:
`import *` makes it harder for `pylint`/`flake8` to statically analyze the code.